### PR TITLE
Apply PR #25690 to 1.7.8.0-build

### LINKED ISSUE #25680: BO - SEO and Webservice Parameters pages not saved in Multistore
#### Describe the bug

When in Multistore, in shop or group of shops context, the parameters are not saved on both SEO & URLS and Webservice pages.

#### Expected behavior

Parameters should be saved in every context in Multistore on these pages.

#### Steps to Reproduce

1. Go to Shop Parameters > Traffic & SEO > SEO & URLs or to Advanced Parameters > Webservice pages
2. Change any parameter (for example : "Accented URL" or "Route to products" / "Enable PrestaShop's webservice"), save
3. Click on the menu to to reload the page (without hitting F5 key)
4. Check the modified parameters : The changes were not saved.

**Screenshots**

https://drive.google.com/file/d/1seW68XZPCbuBoc9gnexE9SKOlwlAHvj2/view

#### Additional information

* PrestaShop version: 1.7.8.0 RC1 B3
* PHP version: N/A

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Apply PR #25690 to 1.7.8.0-build
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #25680
| How to test?      | This PR has already been QA approved: #25690
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25875)

### CODE REVIEW

**Review by atomiix — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]
Ok for this one However since @eternoendless is back I still have strong doubts about the deprecation notice triggered manually

We advise to force the third parameter but even in the core code it is almost never used, I get the feeling that this notice a bit anticipated and we should wait for next version to be sure about the interface we are expecting and if this parameter does indeed become mandatory

TLDR; I would remove the deprecation notice

**Comment by Robin-Fischer-PS** [comment]
No need for QA (except if there is a change between the original PR and this one).

**Comment by eternoendless** [comment]
As discussed, single shop must be treated as a special case of multishop, therefore the depreciation notice is adequate to prepare developers for when the parameter will be enforced.

**Comment by eternoendless** [comment]
Thank you @matthieu-rolland
